lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriterOptions.java

```diff
@@ -13,52 +13,98 @@
  */
 package io.trino.parquet.writer;
 
+import com.google.common.collect.ImmutableSet;
 import io.airlift.units.DataSize;
 import org.apache.parquet.hadoop.ParquetWriter;
 
+import java.util.Set;
+
+import static com.google.common.base.Preconditions.checkArgument;
 import static java.lang.Math.toIntExact;
 import static java.util.Objects.requireNonNull;
 
 public class ParquetWriterOptions
 {
     private static final DataSize DEFAULT_MAX_ROW_GROUP_SIZE = DataSize.ofBytes(ParquetWriter.DEFAULT_BLOCK_SIZE);
     private static final DataSize DEFAULT_MAX_PAGE_SIZE = DataSize.ofBytes(ParquetWriter.DEFAULT_PAGE_SIZE);
+    private static final double DEFAULT_BLOOM_FILTER_FPP = 0.05;
```

Comment:
How did you choose this number?

Reply:
The ORC writer options specifies `0.05 ` as the default probability as well: https://github.com/trinodb/trino/blob/fe608f2723842037ff620d612a706900e79c52c8/lib/trino-orc/src/main/java/io/trino/orc/OrcWriterOptions.java#L52

Reply:
We should look at what the official parquet writer uses, too.

Reply:
It looks like Parquet is internally using FPP of 0.01.

https://github.com/apache/parquet-mr/pull/757/files#diff-db16cfb763bbdc686a577d12c1ccbbc157c917f76b81c06361cfe572f8f337baR59

We can use that value instead too.

Reply:
Let's go with that, so under equivalent conditions, Trino will produce similar parquet files to what other engines would produce.